src/components/implementation/no_interface/cFE_booter/cFE_emu_support.c

```diff
@@ -0,0 +1,117 @@
+#include <stdio.h>
+
+#include <cos_component.h>
+#include <cos_debug.h>
+#include <cos_types.h>
+
+#include <memmgr.h>
+
+#include <cfe_error.h>
+
+#include <cFE_emu.h>
+
```

Comment:
Too many blank lines in header inclusion.

Reply:
We follow the std => cos => interface => cFE => internal header convention

Reply:
It would be better to have that as comments, so it's followed by others who change these files. 
Like,
```
/* standard includes */
xxx
...
/* composite includes */
...
/* interface includes */
...
/* cFE includes */
...
```
These undocumented groupings may not be followed otherwise.

Reply:
It's just a convention we follow. Plus, that's like 4 comments on headers per file. That seems pretty extra, since most of composite groups headers similarly without comments.